### PR TITLE
Info about support for database on non USB / SD devices

### PR DESCRIPTION
**Pull request by fkbreitl**

### CODE REVIEW

File: src/library/rekordbox/rekordboxfeature.cpp [hunk 1]
```diff
@@ -1279,10 +1279,10 @@ TreeItemModel* RekordboxFeature::getChildModel() {
 QString RekordboxFeature::formatRootViewHtml() const {
     QString title = tr("Rekordbox");
     QString summary = tr(
-            "Reads databases for Pioneer CDJ / XDJ players. <br/>"
-            "The database has to be exported to a USB or SD device <br/>"
-            "with a FAT or HFS file system using the Rekordbox Export mode. <br/>"
-            "Not supported are databases that have been moved to an external device via <br/>"
+            "Reads databases exported for Pioneer CDJ / XDJ players with the Rekordbox Export mode. <br/>"
```
Comment (ronso0) [inline src/library/rekordbox/rekordboxfeature.cpp:1282]:
```suggestion
            "Reads databases exported for Pioneer CDJ / XDJ players using the Rekordbox Export mode. <br/>"
```

File: src/library/rekordbox/rekordboxfeature.cpp [hunk 2]
```diff
@@ -1279,10 +1279,10 @@ TreeItemModel* RekordboxFeature::getChildModel() {
...
+            "Rekordbox can only export to USB or SD devices with a FAT or HFS file system. <br/>"
+            "Mixxx reads a database from any device that contains the database folders (PIONEER and Contents). <br/>"
```
Comment (ronso0) [inline src/library/rekordbox/rekordboxfeature.cpp:1284]:
```suggestion
            "Mixxx can read a database from any device that contains the database folders (PIONEER and Contents). <br/>"
```

**Review by ronso0 — COMMENTED** [review]
some suggestions.
Though I'm not a native speaker either ;)

**Review by fkbreitl — COMMENTED** [review]
fixed pre-commit code style issues

File: src/library/rekordbox/rekordboxfeature.cpp [hunk 3]
```diff
@@ -1279,10 +1279,14 @@ TreeItemModel* RekordboxFeature::getChildModel() {
...
-            "Not supported are databases that have been moved to an external device via <br/>"
+            "Reads databases exported for Pioneer CDJ / XDJ players using "
+            "the Rekordbox Export mode. <br/>"
+            "Rekordbox can only export to USB or SD devices with a FAT or "
+            "HFS file system. <br/>"
+            "Mixxx can read a database from any device that contains the "
+            "database folders (PIONEER and Contents). <br/>"
+            "Not supported are Rekordbox databases that have been moved to "
+            "an external device via <br/>"
             "Preferences > Advanced > Database management. <br/>"
```
Comment (Holzhaus) [inline src/library/rekordbox/rekordboxfeature.cpp:1290]:
```suggestion
            "Reads databases exported for Pioneer CDJ / XDJ players using "
            "the Rekordbox Export mode.<br/>"
            "Rekordbox can only export to USB or SD devices with a FAT or "
            "HFS file system.<br/>"
            "Mixxx can read a database from any device that contains the "
            "database folders (<tt>PIONEER</tt> and <tt>Contents</tt>).<br/>"
            "Not supported are Rekordbox databases that have been moved to "
            "an external device via<br/>"
            "<i>Preferences > Advanced > Database management</i>.<br/>"
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you.

**Comment by Be-ing** [comment]
Can you rebase this onto the 2.3 branch?

**Comment by fkbreitl** [comment]
Can I do it on Github? I have no local repository...

**Comment by Be-ing** [comment]
I don't think so. It's going to be pretty hard to contribute if you're only relying on the GitHub web interface instead of using Git locally.

**Comment by fkbreitl** [comment]
I have it for other work but didn't think I would need it for changing this little text.

**Comment by fkbreitl** [comment]
What are the git commands to do this?

**Comment by Be-ing** [comment]
If you haven't already, [configure the upstream remote as described on the wiki](https://github.com/mixxxdj/mixxx/wiki/using-git#set-up-git). Then
```
git checkout patch-1 # switch to your branch
git rebase -i upstream/2.3
```
This will bring up a big list of commits to rebase. Delete all the commits that aren't yours, then save the file and Git will do the rebase. Then
```
git push --force
```
https://git-scm.com/book/en/v2/Git-Branching-Rebasing
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by Be-ing** [comment]
Thank you.

**Pull request merged**